apps/webapp/app/components/navigation/SideMenu.tsx

```diff
@@ -265,16 +268,14 @@ export function SideMenu({ user, project, organization, organizations }: SideMen
                 </div>
               </DialogContent>
             </Dialog>
-          ) : (
-            <SideMenuItem
-              name="Join our Discord"
-              icon={DiscordIcon}
-              to="https://trigger.dev/discord"
-              data-action="join our discord"
-              target="_blank"
-            />
           )}
-
```

Comment:
Paying users get the option of both Discord and Slack. Before it was only Slack if they're paying, which sucked because some people were happily using Discord.